tests/Makefile

```diff
@@ -45,7 +45,7 @@ operator-local:
 		--set=newRelicHost=${NEW_RELIC_HOST} \
 		--set=controllerManager.manager.image.pullPolicy=Never \
 		--set=controllerManager.manager.image.repository=e2e/k8s-agents-operator \
-		--set=controllerManager.manager.image.tag=e2e
+		--set=controllerManager.manager.image.version=e2e
```

Comment:
There's an identical issue in the section above on line 32. Can you fix that as well?

`controllerManager.manager.image.tag` -> `controllerManager.manager.image.version`

Reply:
Ooops sorry for missing this comment. Thanks for making this change